.github/workflows/references.yml

```
@@ -0,0 +1,32 @@
+name: Generate References
+on:
+  push:
+    branches:
+      - main
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Python Deps.
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+
+      - name: Create BibTeX from readme.md
+        run: |
+          cd bibtex
```

Do we need to [make sure the folder exists](https://stackoverflow.com/questions/793858/how-to-mkdir-only-if-a-directory-does-not-already-exist) first?
```suggestion
	  mkdir -p bibtex
          cd bibtex
```